src/libcore/intrinsics.rs

```diff
@@ -278,17 +278,157 @@ extern "rust-intrinsic" {
     /// Moves a value out of scope without running drop glue.
     pub fn forget<T>(_: T) -> ();
 
-    /// Unsafely transforms a value of one type into a value of another type.
+    /// Reinterprets the bits of a value of one type as another type. Both types
```

Comment:
The first paragraph needs to be a single sentence as it's used as a short summary on the module page and search results.